### PR TITLE
Wrote a blogpost about login.testrun.org

### PR DESCRIPTION
**Pull request by ghost**
I wrote a quick blogpost about our most recent innovation.

### CODE REVIEW

**Comment by ghost** [comment]
Alright, I'm done so far. @Simon-Laux @pabzm do you want to take a look at the blogpost before we publish it? https://staging.delta.chat/223/en/2019-12-13-deltachat-identity

**Comment by pabzm** [comment]
Nice! It reads great, thanks!

Unfortunately I'm entangled in other things, so just two notes now:

1. The login works with the iOS-app too. We publish the "testflight"-link on <https://get.delta.chat>, so I'd guess we can speak about it in the blog post as well?
2. I would prefer if the text could state a little clearer that this is an experiment. The first mention of it being a "prototype" is in the third images caption. If you agree and think that publishing this is not pressuring, I could try to to some text work in the next days.
But I don't want to police the tone either, if you disagree or want to publish sooner go ahead.

File: _posts/2019-12-13-deltachat-identity.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: Login to our Forum With DeltaChat Identity - Making Authentication Easy With a Simple QR Code
```
Comment (hpk42) [inline _posts/2019-12-13-deltachat-identity.md:2]: There should be "experiment" in the title already, i guess ...

File: _posts/2019-12-13-deltachat-identity.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+author: compl4xx
+---
+
+
+**What if you could login to a forum, cloud account, or social media platform
+by scanning a simple QR code? Delta Chat makes it possible.**
```
Comment (hpk42) [inline _posts/2019-12-13-deltachat-identity.md:8]: it's not really about the QR code which is just the means to achieve what i'd maybe call decentralized authentication.

File: _posts/2019-12-13-deltachat-identity.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+Imagine you need help with Delta Chat, found a bug, or want to propose a
+feature. You go to our forum, and want to open a topic to discuss it with the
+others from the community. But first you need to create an account, make up a
+password, wait for a verification mail - it's just too much of an effort.
+
+If you have a [GitHub](https://github.com/deltachat/) account, like many Open
+Source developers do, you can click on "Login with GitHub". It's the same as
+"Login with Facebook, Twitter, etc." on other platforms, which we don't offer
+for privacy reasons. But now you can also choose **"Login with DeltaChat
```
Comment (hpk42) [inline _posts/2019-12-13-deltachat-identity.md:26]: Let's keep using "Delta Chat", i.e a space.

File: _posts/2019-12-13-deltachat-identity.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+*Now you are a part of the Delta Chat community forum.*
+
+
+## What's the innovation behind this?
+
+Most of this is not very new. [OAuth2](https://oauth.net/2/) is an established
+standard and has been around for a while. "Login with DeltaChat" uses it just
+as well as "Login with GitHub".
+
+The main innovation is the **QR code** - you can use your phone to login
```
Comment (hpk42) [inline _posts/2019-12-13-deltachat-identity.md:81]: don't have a nice catchy phrase but ... The main innovation is that you can use your e-mail account and self-controlled cryptographic identity to authenticate a web-session.

**Review by hpk42 — CHANGES_REQUESTED** [review]
made a few comments.  Thanks for kicking this off :) 

i might be able to go over the text to streamline a few things, also mention the cryptographic background more.  it's too hard trying to suggest the according changes through github reviews.

File: _posts/2019-12-13-deltachat-identity.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+With this authentication method, the E-Mail address of the contact is available
+directly, which is used by almost every internet service for identification.
+The biggest difference between Delta Chat and other messengers has always been
+that we can use the advantages of the E-Mail network.
+
+For now we only set it up with our Discourse Forum - setting it up for other
+[Discourse](https://www.discourse.org/) instances is easy though. *(If you want
+to set it up for your forum, write me a message:
+[compl4xx@testrun.org](mailto:compl4xx@testrun.org).)* But in theory, it could
```
Comment (Simon-Laux) [inline _posts/2019-12-13-deltachat-identity.md:93]: set up login via login.testrun.org or setup your own login provider?

**Review by Simon-Laux — CHANGES_REQUESTED** [review]
besides that it looks good IMO

**Comment by hpk42** [comment]
This is all good, but i think it's interferecing timing wise too much with doing apk 990 release now. Putting ti to the new "Resurrection" project.

**Comment by hpk42** [comment]
not sure why you re-open it right now -- this is still interfering timing-wise as we are just trying to push out a gplay blog post and need to fix some bugs and do a new apk 1.0.1.

**Pull request closed without merging**